akka-remote/src/test/scala/akka/remote/artery/SerializationErrorSpec.scala

```diff
@@ -41,7 +41,7 @@ class SerializationErrorSpec extends ArteryMultiNodeSpec(ArterySpecSupport.defau
       remoteRef ! "ping"
       expectMsg("ping")
 
-      EventFilter[java.io.NotSerializableException](start = "Failed to serialize message", occurrences = 1).intercept {
+      EventFilter[Exception](start = "Failed to serialize message", occurrences = 1).intercept {
```

Comment:
why not keep `NotSerializableException`?

Reply:
Because we got a different exception

Reply:
> we got a different exception

regression?

Should the test then be explicit about that other exception?

Reply:
Thanks for keeping me honest ;)

This test was still throwing `java.io.NotSerializableException` on Scala 2.13, but not on Scala 3.

This turned out to be because the message is an `object`, which is `java.io.Serializable` on Scala 3, but not on Scala 2.13, leading to a different failure mode. Changing it to a `class`, which is not `java.io.Serializable` on either Scala version, makes it behave the same on both versions again.